kratos/includes/initial_state.h

```diff
@@ -44,7 +42,16 @@ class KRATOS_API(KRATOS_CORE) InitialState
     public:
         ///@name Type Definitions
         ///@{
-        typedef std::size_t SizeType;
+
+        // Size type definition
+        using SizeType = std::size_t;
+
+        /// Pointer definition of NodeSearchUtility
+        KRATOS_CLASS_INTRUSIVE_POINTER_DEFINITION(InitialState);
```

Comment:
First of all, why is this file changed in this PR? I don't see it getting used anywhere. Also, copypasta:

```suggestion
        /// Pointer definition of InitialState
        KRATOS_CLASS_INTRUSIVE_POINTER_DEFINITION(InitialState);
```